validation/helpers.py

```diff
@@ -11,6 +11,8 @@
 from django.conf import settings
 from django.utils.http import urlencode
 
+import validation
```

Comment:
Oh lawd, why does this need to be imported?

Reply:
It doesn't! Good catch